frontend/app/components/admin/setting-multiselect-with-create.js

```diff
@@ -0,0 +1,75 @@
+import Component from "@glimmer/component";
+import { inject as service } from "@ember/service";
+import { action } from "@ember/object";
+import { tracked } from "@glimmer/tracking";
+
+export default class SettingsComponent extends Component {
+  @service fetchService;
+  @service store;
+  @tracked selectedOptions = [];
+
+  get settingRecord() {
+    return this.store.peekAll(this.args.settingModel).firstObject;
+  }
+
+  constructor() {
+    super(...arguments);
+
+    if (!this.args.selectedOptions) {
+      this.store.findAll(this.args.settingModel).then((settings) => {
+        const setting = settings.firstObject;
+        if (this.args.options && this.args.options.length) {
+          this.selectedOptions = this.args.options.filter((option) => {
+            return setting.value.includes(option.value);
+          });
+        } else {
+          this.selectedOptions = setting.value.map((value) => {
+            return { label: value, value: value };
+          });
+        }
+      });
+    } else {
+      this.selectedOptions = this.args.selectedOptions;
+    }
+  }
+
+  @action
+  updateSetting(selected) {
+    this.selectedOptions = selected;
+    this.settingRecord.value = selected.map((option) => {
```

Comment:
![image](https://user-images.githubusercontent.com/655627/121996190-6f462580-cda8-11eb-8b95-81b32872e3ee.png)

Reply:
always check browser console for errors

Reply:
those errors popped up because the settings were not available in db! had to reseed